api/lb/v1/lb_utils.go

```diff
@@ -87,3 +87,70 @@ func waitForLb(timeout *time.Duration, retryInterval *time.Duration, getLB func(
 	}
 	return lb.(*LB), nil
 }
+
+// ZonedAPIWaitForLBPNRequest is used by WaitForLBPN method.
+type ZonedAPIWaitForLBPNRequest struct {
+	LBID          string
+	PNID          string
```

Comment:
Why do you need to wait for PN one by one? Why not wait for all PNs to be up everytime?

Reply:
Today we only support one pn but I can adapt this for future upgrades